server/channels/api4/channel_test.go

```diff
@@ -859,14 +859,23 @@ func TestGetPublicChannelsForTeam(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, channels, 4, "wrong path")
 
-	for i, c := range channels {
+	var foundPublicChannel1, foundPublicChannel2 bool
+	for _, c := range channels {
 		// check all channels included are open
 		require.Equal(t, model.ChannelTypeOpen, c.Type, "should include open channel only")
 
 		// only check the created 2 public channels
-		require.False(t, i < 2 && !(c.DisplayName == publicChannel1.DisplayName || c.DisplayName == publicChannel2.DisplayName), "should match public channel display name")
```

Comment:
The two public channels weren't always returned first in the set of four channels on my machine.